### PR TITLE
Fix lead options assignee

### PR DESCRIPTION
**Pull request by bewakes**

### CODE REVIEW

File: apps/lead/views.py [hunk 1]
```diff
@@ -274,8 +272,8 @@ def get(self, request, version=None):
             {
                 'key': user.id,
                 'value': user.profile.get_display_name(),
-            } for user in User.objects.prefetch_related('profile').filter(
-                project_usergroup_filter
+            } for user in _filter_users_by_projects_memberships(
```
Comment (thenav56) [inline apps/lead/views.py:275]: Adding prefetch_related will make sense.

File: apps/lead/views.py [hunk 2]
```diff
@@ -385,9 +376,7 @@ def post(self, request, version=None):
             # Dynamic Options
 
             'lead_groups': LeadGroup.objects.filter(project_filter, id__in=lead_groups_id).distinct(),
-            'members': (
-                User.objects.filter(id__in=members_id) if len(members_id) else User.objects
-            ).filter(project_usergroup_filter).prefetch_related('profile').distinct(),
+            'members': _filter_users_by_projects_memberships(members_qs, projects).distinct(),
```
Comment (thenav56) [inline apps/lead/views.py]: Adding prefetch_related will make sense.

File: apps/lead/views.py (hunk 1 above)
Comment (bewakes) [reply apps/lead/views.py:275]: done

File: apps/lead/views.py (hunk 2 above)
Comment (bewakes) [reply apps/lead/views.py]: done

**Review by thenav56 — APPROVED** [review]

**Pull request merged**